src/main/java/com/team4/testingsystem/controllers/UsersController.java

```diff
@@ -17,17 +20,24 @@
 @RestController
 public class UsersController {
     private final UsersService usersService;
+    private final TestsService testsService;
 
     @Autowired
-    public UsersController(UsersService usersService) {
+    public UsersController(UsersService usersService, TestsService testsService) {
         this.usersService = usersService;
+        this.testsService = testsService;
     }
 
     @ApiOperation("Get list of all coaches in the system")
     @GetMapping("/coaches")
     public List<UserDTO> getCoaches() {
-        return usersService.getUsersByRole(Role.COACH).stream()
-                .map(UserDTO::new)
+        return convertToDTO(usersService.getUsersByRole(Role.COACH));
+    }
+
+    @GetMapping("/employees")
+    public List<UserDTO> getAllUsers() {
+        return testsService.getUsersWithAssignedTests().stream()
```

Comment:
Let's rename the method to `getAllUsersAndAssignedTests`.
Current name looks like you're going to return only users, that have assigned tests